src/ipyvizzu/templates/ipyvizzu.js

```diff
@@ -124,6 +129,20 @@ if (!window.IpyVizzu) {
       });
     }
 
+    control(element, method, prevId, lastId, ...params) {
```

Comment:
It seems to me that we don't need to link these two promises together here. I would
do this in the `animate()` method:
```
this.controls[animId] = chart.activated;
```
and this in the `control()` method:
```
if(IpyVizzu.nbconvert) IpyVizzu._hide(element);
this.controls[lastId].then(control => {
...
  control[method]();
});